core/server/apps/subscribers/lib/views/subscribe.hbs

```diff
@@ -25,7 +25,7 @@
             <div class="gh-flow">
                 <header class="gh-flow-head">
                     <nav class="gh-flow-nav">
-                        <a href="{{@blog.url}}" class="gh-flow-back"><i class="icon-arrow-left"></i> Back</a>
+                        <a href="{{@blog.url}}" class="gh-flow-back" onclick="window.history.go(-1); return false;"><i class="icon-arrow-left"></i> Back</a>
```

Comment:
`subscribed_url` is set when you are on a post page and you decide to subscribe, then you get redirected to the subscriber page.
`subscribed_url` is not set when you navigate manually to subscriber page or when you refresh. (in this case we route to alternative site `blog.url`)